app/presenters/licence_details_presenter.rb

```diff
@@ -19,6 +19,12 @@ def licence_authority_specific?
     !licence_details["location_specific"]
   end
 
+  def uses_licensify?
+    return false unless interaction
```

Comment:
Should this be `unless action`?  It's not 100% clear to me why we guard against a missing `interaction` when we don't then use it later in the method.

Reply:
interaction is what action is initialized with

```
  def action
    return nil unless interaction
    raise RecordNotFound unless authority["actions"].keys.include?(interaction)

    interaction
  end
```

It is better to just check for action so it's clearer so I'll do that. I was trying to avoid some unnecessary checks.